packages/shared/src/graphql/squads.ts

```diff
@@ -243,6 +243,19 @@ export const SQUAD_STATIC_FIELDS_QUERY = gql`
   }
 `;
 
+export type SquadStaticData = Pick<
+  Squad,
+  'id' | 'name' | 'public' | 'description' | 'image'
+>;
+
+export const getSquadStaticData = async (
+  handle: string,
+): Promise<SquadStaticData> => {
+  const res = await gqlClient.request(SQUAD_STATIC_FIELDS_QUERY, { handle });
+
+  return res.source;
+};
```

Comment:
Just moved it as a generic function, I believe this is used somewhere else, too, but I didn't want to change it here as this is already a big PR.